odoo/addons/base/models/ir_cron.py

```diff
@@ -320,5 +320,12 @@ def try_write(self, values):
 
     @api.model
     def toggle(self, model, domain):
+        # Prevent deactivated cron jobs from being re-enabled through side effects on
+        # neutralized databases. Invalidate the parameter cache so the neutralization
+        # applies without restarting the database.
+        self.env['ir.config_parameter'].clear_caches()
+        if self.env['ir.config_parameter'].sudo().get_param('is_neutralized'):
```

Comment:
would you mind making the ICP name slightly more specific, as if it was in the "database" namespace? ICP keys are a bit messy, and I think it would make the purpose of the param slightly more self-explaining.

```suggestion
        if self.env['ir.config_parameter'].sudo().get_param('database.is_neutralized'):
```